docs/extension-development.md

```diff
@@ -57,7 +57,7 @@ That's it; the setup should now be fully functional.
 
 Now, when we update a package in the `mono` repo, it doesn't automatically move the changes to the `extension`'s `node_modules`. To do this, run the following command in the extension each time you update a package in the monorepo:
```

Comment:
As an FYI, we thought about moving the `extension` into the `mono` repo at some stage but it has a more complicated CI release setup. 

It could be good to eventually migrate it over there if we can maintain the release process

Reply:
Yeah, @kyranjamie and I discussed this - one advantage he brought up having it external is having to consume our packages as truly external which potentially increases their quality, though I see plenty of advantages to bringing `extension` into `mono`